content/docs/command-reference/plots/show.md

```diff
@@ -1,58 +1,72 @@
 # plots show
 
-Generate a plot image from from a [plot metrics](/doc/command-reference/plots)
-file.
+Generate [plot](/doc/command-reference/plots) from a metrics file.
 
 ## Synopsis
 
 ```usage
-usage: dvc plots show [-h] [-q | -v] [-t [TEMPLATE]] [-f FILE]
-                     [-s SELECT] [-x X] [-y Y] [--stdout]
-                     [--no-csv-header] [--no-html] [--title TITLE]
-                     [--xlab XLAB] [--ylab YLAB] [datafile]
+usage: dvc plots show [-h] [-q | -v] [-t <name_or_path>] [-x <field>]
+                      [-y <field>] [--no-header] [--title <text>]
+                      [--x-label <text>] [--y-label <text>] [-o <path>]
+                      [--show-vega]
+                      [targets [targets ...]]
 
 positional arguments:
-  datafile              Metrics file to visualize
+  targets               Metric files to visualize.
+                        Shows all plots by default.
 ```
 
 ## Description
 
 This command provides a quick way to visualize metrics such as loss functions,
-AUC curves, confusion matrices, etc. Please see `dvc plots` for information on
-the supported data formats and other relevant details about DVC plots.
+AUC curves, confusion matrices, etc. All plots defined in `dvc.yaml` are used by
```

Comment:
All plot files are used ... ?

Reply:
That's what the usage says last time I checked. I'm also not sure how this works though, changes to core keep happening too so we can probably follow up on this in #1414 — added a bullet